indico/modules/categories/templates/display/calendar.html

```diff
@@ -0,0 +1,40 @@
+{% from 'forms/_form.html' import simple_form %}
+
+<div class="category-container">
+    <div class="category-header">
+        <div class="category-statistics center">
+            <span class="right">
+                <a class="i-button icon-prev" style="margin-top:0.4em;" href="{{ category.url }}">
+                    {%- trans %}Category page{% endtrans -%}
+                </a>
+            </span>
+            <div class="category-title" style="margin-left:0px; text-align:left;">
+                {{ category.title }}&nbsp;
+                <span style="font-style: italic; font-size: 0.8em;">({% trans %}Calendar{% endtrans %})</span>
+            </div>
+        </div>
+    </div>
+    <div class="category-content-wrapper category-statistics center" style="text-align: left;">
+        <div style="margin: 0 auto;width: 500px;">
```

Comment:
I guess you just didn't want to bother moving the inline CSS to files, right? That's fine, I just hope you didn't write these lines 😝